packages/x-charts/src/BarChart/BarElement.tsx

```diff
@@ -3,7 +3,7 @@ import composeClasses from '@mui/utils/composeClasses';
 import { SlotComponentProps } from '@mui/base';
 import { useSlotProps } from '@mui/base/utils';
 import generateUtilityClass from '@mui/utils/generateUtilityClass';
-import { styled } from '@mui/material/styles';
+import { styled } from '@mui/system';
```

Comment:
Have you considered using scoped imports the same way we decided on avoiding the root imports from `@mui/material`? 🤔 
It's not a big problem, but still, the `system` isn't a tiny package as `utils` are... 🤔